Makefile

```diff
@@ -46,7 +46,7 @@ test-python:
 # `pl_unit_test.py` has an unfortunate file name - it matches the pattern that
 # pytest uses to discover tests, but it isn't actually a test file itself. We
 # explicitly exclude it here.
-	@python3 -m pytest --ignore graders/python/python_autograder/pl_unit_test.py
+	@python3 -m pytest --ignore graders/python/python_autograder/pl_unit_test.py --cov=apps
```

Comment:
I feel like adding coverage reports should be a separate PR, but that's up to @nwalters512 really.

Reply:
I thought about splitting it out, but this was a 2 line change anyway.

Reply:
Yeah, I'm fine to add this here. I guess we should revisit JS test coverage reports as well?